singer-commons/src/main/java/com/pinterest/singer/loggingaudit/client/LoggingAuditClient.java

```diff
@@ -150,6 +150,20 @@ public void audit(String loggingAuditName, LoggingAuditHeaders loggingAuditHeade
         "");
   }
 
+  /**
+   *  create and enqueue a LoggingAuditEvent if TopicAuditConfig exists for this topic/logName,
+   *  enqueueEnabled is true and there is capacity available in the queue.
+   * @param loggingAuditHeaders
+   * @param messageValid
+   * @param messageAcknowledgedTimestamp
+   * @param skipped
```

Comment:
Could you please rename this variable to make it more descriptive?
`messageSkipped`

It's not clear whether it's referring to a configuration or a state of the message.